warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -2125,7 +2125,7 @@ models:
     columns:
       - name: key
         description: |
-          Synthetic primary key constructed from `source_record_id`, `route_id`, `month`, `year`, and `time_of_day`.
+          Synthetic primary key constructed from `source_record_id`, `route_id`, `month`, `year`, `day_type`, and `time_of_day`.
```

Comment:
To be consistent with the contents of the SQL query, can the docs be updated to note that `route_short_name` and `route_long_Name` are also part of the primary key?

Reply:
Done! Also fixed the typo...should be `route_long_name` and swapped the order to match the order of the columns in the table.